### PR TITLE
STIJ-45: Added styling and implementation for jQuery datepicker.

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/21-atoms/inputs/input-date/_input-date.scss [hunk 1]
```diff
@@ -1,3 +1,161 @@
...
+  background-repeat: no-repeat;
+  background-position: right .625rem center;
+  background-size: 1.25rem;
```
Comment (delrueba) [inline components/21-atoms/inputs/input-date/_input-date.scss]: Can these be optimized to:
background: url('data:image/...') no-repeat right .625rem center;

File: components/21-atoms/inputs/input-date/_input-date.scss [hunk 2]
```diff
@@ -1,3 +1,161 @@
...
+          float: left;
+          // sass-lint:disable-block no-duplicate-properties
+          font-size: .6 * 18px;
+          font-size: .6rem;
+          font-weight: normal;
+          line-height: 1.8rem;
+        }
+      }
+
+
```
Comment (delrueba) [inline components/21-atoms/inputs/input-date/_input-date.scss]: whiteline

File: components/21-atoms/inputs/input-date/input-date.twig [hunk 3]
```diff
@@ -1 +1 @@
-<input type="date" name="date" id="date"  class="{{ modifier }}" />
+<input type="text" name="date" class="datepicker" placeholder="dd/mm/jjjj" />
```
Comment (delrueba) [inline components/21-atoms/inputs/input-date/input-date.twig]: really should be type="date"...

Comment (meirege) [reply components/21-atoms/inputs/input-date/input-date.twig]: As discussed, we keep using text for now.

File: components/31-molecules/building-blocks/gallery/gallery.functions.js [hunk 4]
```diff
@@ -21,6 +21,7 @@
         getCaptionFromTitleOrAlt: false,
         counter: false
       });
+
     }
```
Comment (wim-vantomme) [inline components/31-molecules/building-blocks/gallery/gallery.functions.js]: This /n is not necessary

File: components/_preview.twig [hunk 5]
```diff
@@ -7,6 +7,7 @@
 
     <link media="all" rel="stylesheet" href="{{ '/styleguide/vendor/chosen-js/chosen.min.css'| path }}">
     <link media="all" rel="stylesheet" href="{{ '/styleguide/vendor/lightgallery/dist/css/lightgallery.min.css'| path }}">
+    <link media="all" rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```
Comment (wim-vantomme) [inline components/_preview.twig]: unneeded /n

File: components/_preview.twig [hunk 6]
```diff
@@ -22,7 +23,12 @@
       })(document);
     </script>
 
+    <!-- Load jQuery -->
     <script type="text/javascript" src="{{ '/styleguide/vendor/jquery/dist/jquery.min.js'|path }}"></script>
+
+    <!-- Load jQuery UI -->
+    <script src="https://code.jquery.com/ui/1.12.0/jquery-ui.min.js" integrity="sha256-eGE6blurk5sHj+rmkfsGYeKyZx3M4bG+ZlFyA7Kns7E=" crossorigin="anonymous"></script>
```
Comment (wim-vantomme) [inline components/_preview.twig]: Would we not drop all CDN use?

File: components/21-atoms/inputs/input-date/_input-date.scss [hunk 7]
```diff
@@ -1,3 +1,158 @@
...
+      padding-bottom: .5rem;
+      border: 0;
+
+      .ui-datepicker-title {
+        select {
+          display: block;
+          padding: 0;
+          float: left;
+          // sass-lint:disable-block no-duplicate-properties
+          font-size: .6 * 18px;
```
Comment (wim-vantomme) [inline components/21-atoms/inputs/input-date/_input-date.scss]: Why do we not only use a value in rems?

File: components/21-atoms/inputs/input-date/_input-date.scss [hunk 8]
```diff
@@ -1,3 +1,158 @@
...
+          line-height: 1.8rem;
+        }
+      }
+
+      .ui-icon.ui-icon-circle-triangle-w,
+      .ui-state-focus .ui-icon.ui-icon-circle-triangle-w,
+      .ui-state-hover .ui-icon.ui-icon-circle-triangle-w {
+        width: 0;
+        height: 0;
+        border-top: 10px solid transparent;
```
Comment (wim-vantomme) [inline components/21-atoms/inputs/input-date/_input-date.scss]: I would suggest to use rems for the borders as well so border are proportional with our scaling fonts

File: components/21-atoms/inputs/input-date/_input-date.scss [hunk 9]
```diff
@@ -1,3 +1,158 @@
...
+        border-bottom: 10px solid transparent;
+        background-image: none;
+      }
+
+      .ui-icon.ui-icon-circle-triangle-e,
+      .ui-state-focus .ui-icon.ui-icon-circle-triangle-e,
+      .ui-state-hover .ui-icon.ui-icon-circle-triangle-e {
+        width: 0;
+        height: 0;
+        border-top: 10px solid transparent;
```
Comment (wim-vantomme) [inline components/21-atoms/inputs/input-date/_input-date.scss]: Rems in border?

File: components/21-atoms/inputs/input-date/input-date.binding.js [hunk 10]
```diff
@@ -0,0 +1,8 @@
+(function ($) {
+  'use strict';
+
+  $(document).ready(function () {
+    $('.datepicker').loadDatepicker();
+  });
+
```
Comment (wim-vantomme) [inline components/21-atoms/inputs/input-date/input-date.binding.js]: unneeded /n ?

Comment (meirege) [reply components/21-atoms/inputs/input-date/input-date.binding.js]: JS validator requires a newline at the end of JS files apparently. :)

Comment (wim-vantomme) [reply components/21-atoms/inputs/input-date/input-date.binding.js]: ok

**Pull request merged**